repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectConfiguration.scala

```diff
@@ -3,18 +3,61 @@ package cromwell.pipeline.datastorage.dto
 import java.nio.file.Path
 
 import ProjectFile.pathFormat
-import play.api.libs.json.{ Json, OFormat }
+import cats.data.Validated
+import cromwell.pipeline.model.wrapper.VersionValue
+import play.api.libs.functional.syntax.toInvariantFunctorOps
+import play.api.libs.json.{ Format, Json, OFormat }
 
 case class ProjectFileConfiguration(path: Path, inputs: List[FileParameter])
 
 object ProjectFileConfiguration {
   implicit val projectFileConfigurationFormat: OFormat[ProjectFileConfiguration] = Json.format
 }
 
+final case class ProjectConfigurationVersion(value: VersionValue) extends Ordered[ProjectConfigurationVersion] {
```

Comment:
What's wrong with `PipelineVersion`? Why do we need to copy paste a lot of code?

Reply:
The PipeLineVersion has 3 argumets, but we need only one.